src/mame/taito/taito_f3_v.cpp

```diff
@@ -700,2153 +686,737 @@ u16 taito_f3_state::lineram_r(offs_t offset)
 
 void taito_f3_state::lineram_w(offs_t offset, u16 data, u16 mem_mask)
 {
-#if 0
-	/* DariusGX has an interesting bug at the start of Round D - the clearing of lineram
-	(0xa000->0x0xa7ff) overflows into priority RAM (0xb000) and creates garbage priority
-	values.  I'm not sure what the real machine would do with these values, and this
-	emulation certainly doesn't like it, so I've chosen to catch the bug here, and prevent
-	the trashing of priority ram.  If anyone has information on what the real machine does,
-	please let me know! */
-	/* Update: this doesn't seem to occur anymore, I'll leave this snippet in but commented out.
-	 *         fwiw PC=0x1768a0/0x1768a4 is where the game clears lineram in round D, which is a
-	 *         move.w Dn, (An,D7.w*2) , a kind of opcode that could've been bugged back then.
-	 */
-	if (m_game == DARIUSG)
-	{
-		if (m_f3_skip_this_frame)
-			return;
-		if (offset == 0xb000 / 2 && data == 0x003f)
-		{
-			m_f3_skip_this_frame = 1;
-			return;
-		}
-	}
-#endif
-
 	COMBINE_DATA(&m_line_ram[offset]);
 }
```

Comment:
Since there’s no additional code here, can the line RAM be installed directly now, avoiding the trampoline?

Reply:
yes
Should we also remove the unnecessary read handlers? (most of the `_r` functions)

Reply:
sorry, i'm a little confused how to do this with the 16 bit data width.

Reply:
Right, those are 16-bit handlers for being implicitly reused by `2mindril`. The real fix would be to actually convert the whole block to a device, something imo out of scope with this PR.